main.tf

```diff
@@ -56,28 +58,28 @@ data "aws_ami" "nomad_consul" {
 # ---------------------------------------------------------------------------------------------------------------------
 
 module "servers" {
-  source = "github.com/hashicorp/terraform-aws-consul//modules/consul-cluster?ref=v0.3.1"
+  source = "github.com/hashicorp/terraform-aws-consul//modules/consul-cluster?ref=v0.7.0"
 
   cluster_name  = "${var.cluster_name}-server"
-  cluster_size  = "${var.num_servers}"
+  cluster_size  = var.num_servers
   instance_type = "t2.micro"
 
   # The EC2 Instances will use these tags to automatically discover each other and form a cluster
-  cluster_tag_key   = "${var.cluster_tag_key}"
-  cluster_tag_value = "${var.cluster_tag_value}"
+  cluster_tag_key   = var.cluster_tag_key
+  cluster_tag_value = var.cluster_tag_value
 
-  ami_id    = "${var.ami_id == "" ? data.aws_ami.nomad_consul.image_id : var.ami_id}"
-  user_data = "${data.template_file.user_data_server.rendered}"
+  ami_id    = var.ami_id == "" ? data.aws_ami.nomad_consul.image_id : var.ami_id
```

Comment:
This will be the last one of these I call out. You can `grep` for the rest :)

Reply:
:-)

Reply:
resolved